src/astro/databases/base.py

```diff
@@ -139,10 +135,11 @@ def drop_table(self, table: Table) -> None:
     # ---------------------------------------------------------
     def load_file_to_table(
         self,
-        source_file: str,
+        source_file: File,
```

Comment:
:see_no_evil: I thought this was part of this ticket:
https://github.com/astro-projects/astro/issues/330

I'm checking if this PR covers all necessary changes - it may well close 330